rails_event_store/spec/middleware_spec.rb

```diff
@@ -1,42 +1,77 @@
 require 'spec_helper'
+require 'action_controller/railtie'
+require 'rails_event_store/railtie'
+require 'securerandom'
 require 'rails_event_store/middleware'
 require 'rack/lint'
 
 module RailsEventStore
   RSpec.describe Middleware do
-    specify 'lint' do
-      request = ::Rack::MockRequest.new(::Rack::Lint.new(Middleware.new(
```

Comment:
Why have you decided to remove`Rack::Lint` that [verifies](https://www.rubydoc.info/gems/rack/Rack/Lint) Middleware in accordance to `Rack` specification?